kusto/kcsb_test.go

```diff
@@ -166,6 +166,12 @@ func TestGetTokenProviderHappy(t *testing.T) {
 				DataSource:        "https://endpoint/test_tokenprovider_managedidauth2",
 				MsiAuthentication: true,
 			},
+		}, {
```

Comment:
Please also add an API test for WithKubernetesWorkloadIdentity(...)

Reply:
done